plugins/ltac/extraargs.mlg

```diff
@@ -148,7 +148,7 @@ let pr_occurrences = pr_occurrences () () ()
 let pr_gen prc _prlc _prtac c = prc c
 
 let pr_globc _prc _prlc _prtac (_,glob) =
-  let _, env = Pfedit.get_current_context () in
+  let _, env = Pfedit.get_current_context () in (* XXX *)
```

Comment:
@ppedrot What is the `glob` arg type used for?

Reply:
Let me make my question more precise: this argument extension seems rarely used, although we have a lot of stuff dealing with `glob_constr` (e.g. ssr), so I was wondering: are there many ways to take glob constrs in the ML interface for tactics?

Reply:
That's probably an attempt at a poor man's closure, as Ltac doesn't have a first-class support for value-returning functions. I don't really understand why you would need a glob at runtime though. It only seem to be used in rewrite strategies?

Reply:
Ah, no, it's also used by `instantiate`, and from there by `w_refine`. So I guess that you have your answer, that's legacy stuff.